lib/tutorials/getting-started.md

```diff
@@ -67,6 +67,40 @@ Note that we URI encode the name parameter, this is to prevent content injection
 
 The `method` parameter can be any valid HTTP method, array of HTTP methods, or an asterisk to allow any method. The `path` parameter defines the path including parameters. It can contain optional parameters, numbered parameters, and even wildcards. For more details, see [the routing tutorial](/tutorials/routing).
 
+## Creating static pages and content
+
+We've proven that we can start a simple Hapi app with our Hello World application. Next, we'll use a plugin called **inert** to serve a static page. Before you begin, stop the server with **CTRL + C**.
+
+To install **inert** run this command at the command line: `npm install --save inert` This will download inert and add it to your `package.json`, which documents which packages are installed.
+
+Add the following to your `server.js` file:
+
+``` javascript
+server.register(require('inert'), function (err) {
+    if (err) { throw err; }
+});
+
+server.route({
```

Comment:
the idea with calling `server.route` in the callback is to demonstrate that it's generally wise to run code that depends on a plugin within the callback to register so that you can be absolutely sure that plugin exists when your code runs.